src/test/java/org/jacp/service/QuestionServiceTest.java

```diff
@@ -0,0 +1,75 @@
+package org.jacp.service;
+
+import org.jacp.entity.QuestionEntity;
+import org.jacp.error.NoEntityException;
+import org.jacp.repository.QuestionRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import java.util.Optional;
+
+/**
+ * @author saffchen created on 10.08.2023
+ */
+class QuestionServiceTest {
+    @Mock
+    private QuestionRepository questionRepository;
+
+    @InjectMocks
+    private QuestionService questionService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    void get() {
+        QuestionEntity questionEntity = new QuestionEntity();
+        Long questionId = 1L;
+        String problem = "TestProblem";
```

Comment:
Duplicated code can be extracted to some common method.

Reply:
Fixed